code/datums/records.dm

```diff
@@ -68,7 +68,6 @@
 	var/real_rank = "Unassigned"
 	var/rank = "Unassigned"
 	var/age = 0
-	var/sex = "Unknown"
```

Comment:
Keep this in. I shouldn't be guessing someone's actual sex as a doctor.

Reply:
Second this. Mofo also doesn't want people playing straight up trans/traps Tajaran, so some way of knowing when somebody does this would be greatly helpful for reporting purposes.